dist/bundle.js

```diff
@@ -667,9 +667,23 @@ function render(hash, options) {
             }
         });
 
+        // Special case: If a single node was changed, then
+        //    bbox.left == bbox.right, and
+        //    bbox.top == bbox.bottom
+        // In this case, add a little padding to avoid breaking fitBounds
+        var {left, right, top, bottom} = bbox
```

Comment:
It should work on most recent browsers. Better not to use destructuring, just in case.